retroshare-gui/src/gui/gxsforums/CreateGxsForumMsg.cpp

```diff
@@ -425,6 +428,8 @@ void  CreateGxsForumMsg::createMsg()
 		case GxsIdChooser::KnowId:
 		case GxsIdChooser::UnKnowId:
 			msg.mMeta.mAuthorId = authorId;
+			if(!mOrigMsgId.isNull() && authorId != mPosterId)
```

Comment:
I don't understand the reason for the check of mOrigMsgId: it is the ID of the original post author in case the post was already edited, but you may moderate posts that have already been moderated or not. Did I miss anything?!?